hazelcast/src/test/resources/hazelcast-client-test-external.yaml

```diff
@@ -0,0 +1,48 @@
+hazelcast-client:
+  cluster-name: dev
+  network:
+    smart-routing: true
+    redo-operation: false
+    connection-timeout: 5000
+    cluster-members:
+      - 127.0.0.1:5701
+    socket-options:
+      tcp-no-delay: true
+      keep-alive: true
+      reuse-address: true
+      linger-seconds: 3
+      buffer-size: 128
+    socket-interceptor:
+      enabled: false
+    aws:
```

Comment:
IIRC all of those are disabled by default, so maybe we can simplify this config to be as minimal as possible?

Reply:
Ok